rstream/__init__.py

```diff
@@ -1,7 +1,19 @@
-from .amqp import AMQPMessage, amqp_decoder
-from .constants import OffsetType
-from .consumer import Consumer
-from .producer import Producer, RawMessage
+from importlib import metadata
+
+try:
+    __version__ = metadata.version(__package__)
+    __license__ = metadata.metadata(__package__)["license"]
+except metadata.PackageNotFoundError:
+    __version__ = "dev"
+    __license__ = None
+
+del metadata
+
+
+from .amqp import AMQPMessage, amqp_decoder  # noqa: E402
```

Comment:
nit: you can move these imports to the top of the file so you don't have to add `noqa` comments

Reply:
Oh, realized that's not an option because of circular imports, sorry